### PR TITLE
Add restricted seccomp profile to controller SCC

### PR DESCRIPTION
**Pull request by akalenyu**
Signed-off-by: Alex Kalenyuk <akalenyu@redhat.com>

**What this PR does / why we need it**:
With PSA, one of the things we will try to comply with is the seccomp profile:
https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted
Thus, we need our SCC to allow setting this.
Note this is also included in the restricted-v2 SCC:
```bash
$ oc get scc restricted-v2 -o json | jq .seccompProfiles
[
  "runtime/default"
]
```

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/virt-operator/resource/generate/components/scc.go [hunk 1]
```diff
@@ -74,6 +74,9 @@ func NewKubeVirtControllerSCC(namespace string) *secv1.SecurityContextConstraint
 	scc.SELinuxContext = secv1.SELinuxContextStrategyOptions{
 		Type: secv1.SELinuxStrategyRunAsAny,
 	}
+	scc.SeccompProfiles = []string{
+		"runtime/default",
```
Comment (xpivarc) [inline pkg/virt-operator/resource/generate/components/scc.go:78]: Shouldn't it be `RuntimeDefault`?

Comment (akalenyu) [reply pkg/virt-operator/resource/generate/components/scc.go:78]: nope, check out the snippet from a running cluster in the desc

**Review by xpivarc — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**